vercel/data_source_project.go

```diff
@@ -393,6 +393,14 @@ For more detailed information, please see the [Vercel documentation](https://ver
 				Optional:    true,
 				Computed:    true,
 			},
+			"build_machine_type": schema.StringAttribute{
+				Description: "When `on_demand_concurrent_builds` is enabled, choose the build machine: `standard` (4 vCPU, 8 GB) or `enhanced` (8 vCPU, 16GB)",
+				Optional:    true,
+				Computed:    true,
```

Comment:
I think we can add `UseStateForUnknown`, as this setting will never change based on the other settings changing, right? 
Or does disabling `on_demand_concurrent_builds` affect this?

Reply:
edit: i'm a fool, this is the data source.